src/handlers/feature_handlers.cpp

```diff
@@ -6527,13 +6527,17 @@ void aiVehicleInstanceFeatureHandler::DrawHeadlights()
         vec.Subtract(camPosition);
 
         float invGlowScale = -ltLight::GlowScale;
+
+        //move the headlight position a bit forward (made by default)
         light->Position.AddScaled(vec, invGlowScale * 0.1f);
 
         if (cfgLightClippingFix.Get() == 2 || cfgLightClippingFix.Get() >= 4)
         {
             lvlSegment segment;
             lvlIntersection intersection;
 
+            //replace the hack above with this one
+            //by this we use the actual headlight position
             light->Position.AddScaled(vec, 0.f);
```

Comment:
I don't think you realise what AddScaled does. https://github.com/Fireboyd78/mm2hook/blob/b39462f76e1324624b304ff21a09ad77b1dd33e5/src/mm2_vector.h#L123-L127
You are scaling vec by 0 before adding it, so you are just adding 0, which is basically doing nothing.

Reply:
I think I do know what it does otherwise I wouldn't use it.